git/git.md

```diff
@@ -1,12 +1,12 @@
 # Git
 
-- [Repositories](#repositories)
+- [Repository names](#repositories)
 - [Working with branches](#working-with-branches)
 - [Commit messages](#commit-messages)
 - [Pull requests](#pull-requests)
 - [Versioning](#versioning)
 
-## Repositories
+## Repository names
```

Comment:
the only (very minor) problem that I can see arising is if `file/to/git#repositories` is linked from elsewhere - but the consumer would still end up on the page (at the top, rather than the linked section) … as the page has the toc - they should be able to 'find their way' :)

Reply:
Ooh good point, ta … I did a search, apparently it's not linked directly from anywhere 👍